health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/NewCampaignCreate/CampaignDetails.js

```
@@ -120,24 +120,56 @@ const CampaignDetails = () => {
       <div className="campaign-details-header">
-        <HeaderComponent className={"date-header"}>{campaignData?.campaignName}</HeaderComponent>
-        <Tag label={campaignData?.campaignName} showIcon={false} className={"campaign-view-tag"} type={"warning"} stroke={true}></Tag>
-        <Tag label={campaignData?.deliveryRules?.[0]?.cycles?.length>1 ? t("HCM_MULTIROUND") : t("HCM_INDIVIDUAL")}  showIcon={false} className={"campaign-view-tag"} type={"monochrome"} stroke={true}></Tag>
+        <div style={{ display: "flex", alignItems: "baseline", gap: "1rem" }}>
+          <HeaderComponent className={"date-header"}>{campaignData?.campaignName}</HeaderComponent>
+          <div
+            className="hover"
+            onClick={() => {
+              history.push(`/${window.contextPath}/employee/campaign/create-campaign?key=2&editName=${true}&id=${campaignData?.id}`);
+            }}
+          >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)